samples/dashboard/demos/climate/demo.js

```diff
@@ -199,30 +204,17 @@ async function setupDashboard() {
                             dataPool
                                 .getStore(city)
                                 .then(store => {
-                                    const data = store.table.modified.getRows(
-                                        void 0, void 0,
-                                        ['time', dataScope]
-                                    );
+                                    dataScope = 'TX';
```

Comment:
If I'm looking at something else (for example rain) and I click on another city I don't get the data I needed (rain).
To me, it is a bit confusing. What do you think?

Reply:
We adjust it when the final design will come ;)